src/server/scripts/EasternKingdoms/SunwellPlateau/boss_muru.cpp

```diff
@@ -393,54 +376,48 @@ class npc_dark_fiend : public CreatureScript
 
         void Initialize()
         {
-            WaitTimer = 2000;
-            InAction = false;
-        }
+            me->SetDisplayId(me->GetCreatureTemplate()->Modelid2);
```

Comment:
this npc needs to use the invisible model, but sometimes he uses the visible one, so is needed to force to use the invisible